README.md

```diff
@@ -12,14 +12,19 @@ or
 yarn add @workos-inc/authkit-react
 ```
 
+## Setup
+
+Add your site URL to the list of allowed origins in the WorkOS dashboard by
+clicking on the "Configure sessions" button of the "Authentication" page.
+
 ## Usage
 
 ```jsx
 import { useAuth, AuthKitProvider } from "@workos-inc/authkit-react";
 
 function Root() {
   return (
-    <AuthKitProvider clientId="client_123456">
+    <AuthKitProvider clientId="client_123456" apiHostname="api.workos.com">
```

Comment:
Since this is the default value, maybe use a non-default as an example:

```suggestion
    <AuthKitProvider clientId="client_123456" apiHostname="your-app.example.com">
```

Reply:
ah good call, i renamed this to `auth.example.com`